.tools/readmes/requirements.txt

```diff
@@ -1,4 +1,4 @@
 jinja2>=3.0.3
 pyyaml>=5.3.1
 typer==0.15.1
-aws-doc-sdk-examples-tools @ git+https://github.com/awsdocs/aws-doc-sdk-examples-tools
+aws-doc-sdk-examples-tools @ git+https://github.com/awsdocs/aws-doc-sdk-examples-tools
```

Comment:
These requirements files look like local changes, should they be in this PR?

Reply:
This and the requirments_freeze.txt now mirror the latest in 
https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/.tools/readmes/requirements.txt
https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/.tools/readmes/requirements_freeze.txt
respectively. So shouldn't make a differen AFAICT